framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -326,6 +321,9 @@ describe('MessageRecoveryCommand', () => {
 			];
 			ccmsEncoded = ccms.map(ccm => codec.encode(ccmSchema, ccm));
 			transactionParams.crossChainMessages = [...ccmsEncoded];
+			// Choose the index such that the position of the ccm in the outbox tree is at terminatedChainOutboxSize,
+			// i.e. just outside of the tree. Note that a 1 has to be prepended to the binary representation of the index, which is done by adding 2**(index.length).
+			// See https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md#proof-serialization.
```

Comment:
> Note that a 1 has to be prepended to the binary representation of the index, which is done by adding 2**(index.length). See https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md#proof-serialization.

I would maybe remove this part of the comment as this is not specific to this test, or move it back to line 65. If you prefer to keep it here, I would at least remove "which is done by adding 2**(index.length)" because it's probably not clear to the reader what `index.length` means.

Reply:
yeah doesn't suit here, I will remove it as its not necessary to put comments in the test and also the comment doesn't really help much here. I have removed this part of the comment to avoid any confusion